low-level/files/opt/seagate/sspl/conf/sspl.post-install.conf.tmpl

```diff
@@ -0,0 +1,26 @@
+cortx:
+  release:
+    product: LR_R2
+server_node:
+  30512e5ae6df9f1ea02327bab45e499d:
+    bmc:
+      ip: 198.120.10.1
+      secret: Admin123
+      user: ADMIN
```

Comment:
maintain order ip, user, secret

Reply:
These templates kv are populated using conf store cli. Though cli commands are sent in expected order, this is the outcome. As discussed, proceeding with as it is.